packages/datadog-plugin-knex/src/index.js

```diff
@@ -1,14 +1,14 @@
 'use strict'
 
-const tx = require('../../dd-trace/src/plugins/util/promise')
+const { wrapThen } = require('../../datadog-instrumentations/src/helpers/promise')
```

Comment:
`knex` should be moved as well since while not a promise it's a thenable and not actually instrumented, only patched.